doc/sphinx/src/using-eos.rst

```diff
@@ -1311,3 +1326,4 @@ will use it.
   an initialized variable, even if it is zero-initialized. Do not pass
   uninitialized memory into this function!
 
+This file was made in part with generative AI.
```

Comment:
If this is policy now, we probably want to reflect it in the contribution guidelines